nvdaHelper/vbufBackends/gecko_ia2/gecko_ia2.cpp

```diff
@@ -273,6 +273,16 @@ bool hasAriaHiddenAttribute(const map<wstring,wstring>& IA2AttribsMap){
 	return (IA2AttribsMapIt != IA2AttribsMap.end() && IA2AttribsMapIt->second == L"true");
 }
 
+std::optional<wstring> getAccDescription(IAccessible2* pacc, VARIANT childID) {
+	std::optional<wstring> desc;
+	CComBSTR rawDesc;
+	if (pacc->get_accDescription(childID, &rawDesc) == S_OK) {
+		desc = rawDesc;
+		SysFreeString(rawDesc);

```

Comment:
This is the problem, the string is being freed twice. It is now a `CComBSTR` which will free on destruction.